src/unix/solarish/solaris.rs

```diff
@@ -189,6 +189,8 @@ pub const PRIV_TPD_UNSAFE: c_uint = 0x0800;
 pub const PRIV_PROC_TPD_RESET: c_uint = 0x1000;
 pub const PRIV_TPD_KILLABLE: c_uint = 0x2000;
 
+pub const POSIX_SPAWN_SETSID: c_short = 0x400;
+
```

Comment:
I didn't see a `solaris.txt`, and I wasn't sure whether a definition being in both `solaris.rs` and `illumos.rs` was enough to put it in `solarish.txt`.